[DOC] Mention CORS is required when external URLs are allowed in Vega (#118862)
elastic/kibana: docs/user/dashboard/vega-reference.asciidoc

*Vega* can load data from any URL. To enable, set `vis_type_vega.enableExternalUrls: true` in `kibana.yml`,
then restart {kib}.

> Please be aware the files served by those external URLs must allow https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS[CORS].

Reviewer: ```suggestion
The files that the external URLs load must allow https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS[CORS].
```

> It is necessary the remote URL has `Access-Control-Allow-Origin` which allows requests from the Kibana URL.

Reviewer: ```suggestion
The remote URL must include `Access-Control-Allow-Origin`, which allows requests from the {kib} URL.
```
